node-ui/src/App.tsx

```diff
@@ -31,6 +31,7 @@ export default function App() {
 
   useEffect(() => {
     setNodeUrlFromQuery(showServerDownPopup);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
To disable warnings but won't be adding suggested variables because of unexpected behaviour of useEffect function.